react/ListPrice.tsx

```diff
@@ -29,7 +29,10 @@ const ListPrice: StorefrontFC<BasicPriceProps> = props => {
   const { taxPercentage } = commercialOffer
   const listPriceWithTax = listPriceValue + listPriceValue * taxPercentage
 
-  if (listPriceValue === sellingPriceValue) {
+  if (
+    listPriceValue === sellingPriceValue ||
+    listPriceValue <= sellingPriceValue
```

Comment:
```suggestion
    listPriceValue <= sellingPriceValue
```
I think this is enough!